README.md

```diff
@@ -45,15 +45,25 @@ npm install @tokens-studio/sd-transforms
 
 > Note: this library is available both in CJS and ESM
 
+## create a file to run custom transformers 
+
 ```js
 const { registerTransforms } = require('@tokens-studio/sd-transforms');
-const StyleDictionary = require('style-dictionary');
+// import style dictionary and the config for it.
+const StyleDictionary = require('style-dictionary').extend(
+  './style-dictionary-config.json',
+);
```

Comment:
I think the transforms etc. should be registered on the StyleDictionary object, globally, not on the StyleDictionary.Core instance that's created by calling StyleDictionary.extend(). Especially if you start doing theming which does multiple .extend() calls (for each theme), it will be best to register them globally on the StyleDictionary object.